org.civicrm.reqangular/test/directives/horizontal-scroll-shadows.directive.spec.js

```diff
@@ -0,0 +1,149 @@
+/* eslint-env amd, jasmine */
+
+define([
+  'common/angular',
+  'common/lodash',
+  'common/angularMocks',
+  'common/directives/horizontal-scroll-shadows.directive'
+], function (angular, _) {
+  'use strict';
+
+  describe('horizontalScrollShadows directive', function () {
+    var $compile, $document, $rootScope, $window, element, scope;
+
+    beforeEach(module('common.directives'));
+    beforeEach(inject(function (_$compile_, _$document_, _$rootScope_, _$window_) {
+      $compile = _$compile_;
+      $document = _$document_;
+      $rootScope = _$rootScope_;
+      $window = _$window_;
+    }));
+
+    describe('when "horizontal-scroll-shadows" attribute is specified', function () {
```

Comment:
there is really no point to this outer, all-wrapping section. If you are testing the directive, of course you're interested in when the directive is applied. Please remove it

Reply:
Done